addons/payment_authorize/models/authorize_request.py

```diff
@@ -4,6 +4,7 @@
 from StringIO import StringIO
 import xml.etree.ElementTree as ET
 from uuid import uuid4
+import logging
```

Comment:
```python
import logging

_logger = logging.getLogger(__name__)

# stuff happens...
_logger.warning('Yikes')
```

otherwise your log will not integrate properly in the odoo logs (formatting, etc.)